test/specs/pruebahumo.test.js

```diff
@@ -0,0 +1,80 @@
+import login from '../pageobjects/Login.page';
+import homePage from '../pageobjects/home.page'
+import formu from '../pageobjects/form.page'
+
+describe('prubahumo', () => {
+  
+    it('login bank', async () => {
```

Comment:
Una convencion que suele utilizarse para la descripcion de test cases es la de estructurarla de acuerdo a "DEBERIA....CUANDO" (o should/when). Hacerlo asi te lleva a enfocar el test en el resultado esperado (lo que **deberia** suceder), **cuando** ejecuto determinado _conjunto de acciones_ con determinado _conjunto de datos_, por lo que la accion y su resultado quedan determinados en el titulo. Luego de eso, definir el test resulta mas facil, ya que el objetivo quedo claro desde el inicio.

En este caso, una buena descripcion usando esa estructura podria ser: `Deberia acceder a la pagina principal cuando se loguea con credenciales validas`